src/aws_signature.erl

```diff
@@ -101,10 +101,10 @@ signature(AccessKeyID, SecretAccessKey, Region, Service, DateTime, Method, URL,
       Method :: binary(),
       URL :: binary(),
       Headers :: headers(),
-      Body :: binary(),
+      Body :: binary() | nil,
```

Comment:
Erlang doesn't use `nil` - _usually_ the preferred atom is `undefined`

Reply:
Thanks, didn't know about that